### PR TITLE
[ja] Translate content/en/docs/concepts/workloads/autoscaling.md into Japanese

### PR DESCRIPTION
**Pull request by kohbis**
… Japanese

close https://github.com/kubernetes/website/issues/46663

## Change List

- add `content/ja/docs/concepts/workloads/autoscaling.md`

/language ja

### CODE REVIEW

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (dipesh-rawat) [inline content/ja/docs/concepts/workloads/autoscaling.md:78]: In this line `{{< feature-gate-description name="InPlacePodVerticalScaling" >}}`, "feature-gate-description" shortcode is not being able to find the file for the “InPlacePodVerticalScaling” feature gate in the Japanese localization. To fix this, we need to localize the English file content in [content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md?plain=1) in Japanese or we can temporarily localize the description and add it directly here, then swap it when all the feature gate description files [(here)](https://github.com/kubernetes/website/tree/main/content/en/docs/reference/command-line-tools-reference/feature-gates) are localized. 
This is just a suggestion, and I will defer to the Japanese localization team for the best course of action.

Comment (dipesh-rawat) [inline content/ja/docs/concepts/workloads/autoscaling.md:78]: In this line {{< feature-gate-description name="InPlacePodVerticalScaling" >}}, "feature-gate-description" shortcode is not being able to find the file for the “InPlacePodVerticalScaling” feature gate in the Japanese localization. To fix this, we need to localize the English file content in [content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/in-place-pod-vertical-scaling.md?plain=1) in Japanese or we can temporarily localize the description and add it directly here, then swap it when all the feature gate description files [(here)](https://github.com/kubernetes/website/tree/main/content/en/docs/reference/command-line-tools-reference/feature-gates) are localized.
This is just a suggestion, and I will defer to the Japanese localization team for the best course of action.

**Review by dipesh-rawat — COMMENTED** [review]
Looks like the PR build preview failed ([View logs (here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/6665b9ff0c9c200008747acb)) with following errors:

> 3:21:30 PM: ERROR [ja] Feature gate details not found for "InPlacePodVerticalScaling" (language code "ja")
3:21:31 PM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
3:21:31 PM: Total in 65530 ms
3:21:31 PM: Error: error building site: "/opt/build/repo/content/ja/docs/concepts/workloads/autoscaling.md:78:1": failed to render shortcode "feature-gate-description": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/feature-gate-description.html:15:25": execute of template failed: template: shortcodes/feature-gate-description.html:15:25: executing "shortcodes/feature-gate-description.html" at <$thisFeatureGate.Content>: can't evaluate field Content in type bool
make: *** [Makefile:43: deploy-preview] Error 1

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (kohbis) [reply content/ja/docs/concepts/workloads/autoscaling.md:78]: @dipesh-rawat 
Thank you for your confirmation.
I will copy the markdown file and translate the content in another PR.

**Comment by Okabe-Junya** [comment]
/hold

Feel free to unhold when https://github.com/kubernetes/website/pull/46749#discussion_r1632428806 solved

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (kohbis) [reply content/ja/docs/concepts/workloads/autoscaling.md:78]: I opened PR https://github.com/kubernetes/website/pull/46755 to fix it.

**Comment by Okabe-Junya** [comment]
/retitle [ja] Translate content/en/docs/concepts/workloads/autoscaling.md into Japanese

**Comment by Okabe-Junya** [comment]
Since #46755 merged, let me unhold

/unhold

Please take a look @atoato88 @kakts to move this PR forward, thank you!

**Comment by atoato88** [comment]
The page(s) to be reviewed:
- https://deploy-preview-46749--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/workloads/autoscaling/

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:22]:
```suggestion
## ワークロードを手動でスケーリングする {#scaling-workloads-manually}
```
We should add the anchor of html as a original `en` title.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:33]:
```suggestion
## ワークロードを自動でスケーリングする {#scaling-workloads-automatically}
```
ditto

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:39]:
```suggestion
### ワークロードを水平方向にスケーリングする {#scaling-workloads-horizontally}
```
ditto

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:47]:
```suggestion
### ワークロードを垂直方向にスケーリングする {#scaling-workloads-vertically}
```
ditto

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:71]:
```suggestion
#### インプレースリサイズの要件 {#requirements-for-in-place-resizing}
```
ditto

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:80]:
```suggestion
### クラスターサイズに基づく自動スケーリング {#autoscaling-based-on-cluster-size}
```
ditto

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:92]:
```suggestion
### イベント駆動型自動スケーリング {#event-driven-autoscaling}
```
ditto.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:99]:
```suggestion
### スケジュールに基づく自動スケーリング {#autoscaling-based-on-schedules}
```
ditto.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:106]:
```suggestion
## クラスターのインフラストラクチャーのスケーリング {#scaling-cluster-infrastructure}
```
ditto.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:26]:
```suggestion
垂直スケーリングの場合、ワークロードのリソース定義を _パッチ適用_ する必要があります。
```
In [original `en` content](https://github.com/kubernetes/website/blob/13cae23279b9a224b5b338384aa16ee92c1346c7/content/en/docs/concepts/workloads/autoscaling.md?plain=1#L28), the "_" syntax is used.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:116]:
```suggestion
  - [StatefulSetのスケール](/ja/docs/tasks/run-application/scale-stateful-set/)
```
We use the link for `ja` content if exists.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:117]:
```suggestion
  - [HorizontalPodAutoscalerウォークスルー](/ja/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/)
```
ditto.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:30]:
```suggestion
- **水平スケーリング**: [Running multiple instances of your app](/ja/docs/tutorials/kubernetes-basics/scale/scale-intro/)
```
We use the link of `ja` content if exists.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:45]:
```suggestion
Deployment用のHorizontalPodAutoscalerを構成する[ウォークスルーチュートリアル](/ja/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough)があります。
```

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [reply content/ja/docs/concepts/workloads/autoscaling.md:78]: #46755 has been merged, we can go forward.
Thanks.

**Review by atoato88 — COMMENTED** [review]
I left comments, please refer those.

File: content/ja/docs/concepts/workloads/autoscaling.md
Comment (atoato88) [inline content/ja/docs/concepts/workloads/autoscaling.md:57]: BTW, the word of "HPA" is weird at here, because this section said the content about "VPA" not "HPA". :thinking:

Comment (atoato88) [reply content/ja/docs/concepts/workloads/autoscaling.md:57]: I know the original `en` content also says "HPA" here.

Comment (atoato88) [reply content/ja/docs/concepts/workloads/autoscaling.md:57]: It's just the comment, not request to change here.

**Comment by kohbis** [comment]
@atoato88 

 thanks for your reviewing, I'd checked and committed suggested changes.

**Review by Okabe-Junya — COMMENTED** [review]
This is not official, but LGTM!

/label tide/merge-method-squash

**Comment by Okabe-Junya** [comment]
/assign nasa9084
for approval

**Review by Okabe-Junya — APPROVED** [review]
LGTM, so please `/unhold` (and merge) if there are no objections within the next 48 hours

（あと2日くらい反対がなかったら `/unhold` してマージしてもらって大丈夫です）

/lgtm
/hold

**Pull request merged**
